### ISSUE TITLE
Chest Remap Issues

### ISSUE THREAD

**Issue by k9lego** [ProtocolSupport/ProtocolSupportPocketStuff#99]
Flooded chests and double chests need to be properly remapped. Great job so far! What you've completed is impressive!

**Comment by 7kasper** [comment]
Yes and after merging with master, its finally possible to do this properly :smile:
